api/turing/api/appcontext.go

```diff
@@ -128,7 +126,7 @@ func NewAppContext(
 
 		batchClusterController, ok := clusterControllers[cfg.BatchEnsemblingConfig.JobConfig.DefaultEnvironment]
 		if !ok {
-			return nil, errors.Wrapf(err, "Failed getting the batch ensembling job controller")
+			return nil, fmt.Errorf("Failed getting the batch ensembling job controller")
```

Comment:
Why switch from the wrapped error?

Reply:
err is actually nil if the code reaches here